src/main.cpp

```diff
@@ -369,44 +371,78 @@ int main(int argc, char* argv[])
     qsrand(static_cast<quint64>(QTime::currentTime().msecsSinceStartOfDay()));
 
     QString homeDirectory = mudlet::getMudletPath(mudlet::mainPath);
-    QString fontDirectory = mudlet::getMudletPath(mudlet::mainFontsPath);
     QDir dir;
     bool first_launch = false;
     if (!dir.exists(homeDirectory)) {
         dir.mkpath(homeDirectory);
         first_launch = true;
     }
 
-    if (!dir.exists(fontDirectory)) {
-        dir.mkpath(fontDirectory);
-    }
 
+#if defined(INCLUDE_FONTS)
     if (show_splash) {
         splash_message.append("Done.\n\nLoading font files... ");
         splash.showMessage(splash_message, Qt::AlignHCenter | Qt::AlignTop);
         app->processEvents();
     }
 
+    QString bitstreamVeraFontDirectory(QStringLiteral("%1/ttf-bitstream-vera-1.10").arg(mudlet::getMudletPath(mudlet::mainFontsPath)));
+    if (!dir.exists(bitstreamVeraFontDirectory)) {
+        dir.mkpath(bitstreamVeraFontDirectory);
+    }
+    QString ubuntuFontDirectory(QStringLiteral("%1/ubuntu-font-family-0.83").arg(mudlet::getMudletPath(mudlet::mainFontsPath)));
+    if (!dir.exists(ubuntuFontDirectory)) {
+        dir.mkpath(ubuntuFontDirectory);
+    }
+
     // The original code plonks the fonts AND the Copyright into the MAIN mudlet
     // directory - but the Copyright statement is specifically for the fonts
-    // so now they all go into a "./fonts/" subdirectory - I note that
+    // so now they all go into "./fonts/" subdirectories - I note that
     // the Debian packager already removes these fonts anyhow as they are
     // already present in a shared form in the OS anyhow so our copy is
-    // ancient and superfluous (they are using 2.37 compared to our 1.10) ...!
-    copyFont(fontDirectory, QLatin1String("COPYRIGHT.TXT"));
-    copyFont(fontDirectory, QLatin1String("RELEASENOTES.TXT"));
-    copyFont(fontDirectory, QLatin1String("README.TXT"));
-    copyFont(fontDirectory, QLatin1String("local.conf"));
-    copyFont(fontDirectory, QLatin1String("Vera.ttf"));
-    copyFont(fontDirectory, QLatin1String("VeraBd.ttf"));
-    copyFont(fontDirectory, QLatin1String("VeraBI.ttf"));
-    copyFont(fontDirectory, QLatin1String("VeraIt.ttf"));
-    copyFont(fontDirectory, QLatin1String("VeraMono.ttf"));
-    copyFont(fontDirectory, QLatin1String("VeraMoBd.ttf"));
-    copyFont(fontDirectory, QLatin1String("VeraMoBI.ttf"));
-    copyFont(fontDirectory, QLatin1String("VeraMoIt.ttf"));
-    copyFont(fontDirectory, QLatin1String("VeraSe.ttf"));
-    copyFont(fontDirectory, QLatin1String("VeraSeBd.ttf"));
+    // superfluous...!
+    // Note that the ubuntu fonts have *just* entered the Unstable "non-free"
+    // Debian version as of Dec 2017:
+    // https://anonscm.debian.org/cgit/pkg-fonts/fonts-ubuntu.git/
+    // but there is a term in the Ubuntu licence that makes them currently (and
+    // for the prior seven years) not quite the right side of the Debian Free
+    // Software Guidelines.
```

Comment:
Why would we need to worry about the status of our dependencies in downstream projects? :thinking: We have a lot downstream projects...

Reply:
Well if we poison our code with stuff they do not want to swallow they either have to cut out the tainted parts or drop the product - by keeping it simple for them to avoid the bits they don't want, it makes it more likely they will continue to package Mudlet rather than taking the other more drastic solution.

Reply:
Just an FYI, Mudlet is not a Debian project...

Reply:
But, sure, this can stay. Until someone else comes along favouring some other distribution and will want to add their downstream stuff too 🤔